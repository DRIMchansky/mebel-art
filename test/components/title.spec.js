import React from 'react'
import renderer from 'react-test-renderer'
import Title from '../../src/components/title'

describe('Title', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<Title />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
