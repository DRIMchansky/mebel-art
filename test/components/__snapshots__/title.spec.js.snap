// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
.c0 {
  color: #212529;
  text-align: center;
  margin: 1.5rem 0;
}

.c0 h1,
.c0 h2,
.c0 h3,
.c0 h4,
.c0 h5,
.c0 h6 {
  font-size: 1.7rem;
  font-weight: 300;
  margin: 0;
}

.c0 span {
  display: block;
  font-size: 1.1rem;
  font-weight: 300;
  margin-top: 0.5rem;
}

.c0 a {
  color: #1c5796;
}

<div
  className="c0"
/>
`;
